script.js

```diff
@@ -0,0 +1,214 @@
+const chat = document.getElementById("chat")
+const nameForm = document.getElementById("name-form")
+const nameInput = document.getElementById("name-input")
+const inputWrapper = document.getElementById("input-wrapper")
+
+let userName = ""
+
+const graphicCathegories = `
+<select id="select">
+<option value="" selected disabled>Click here to choose</option>
+<option value="Web design">Web design</option>
+<option value="Posters">Posters</option>
+<option value="Print ready">Print ready</option>
+</select>
+`
+
+const illustrationCathegories = `
+<select id="select">
+<option value="" selected disabled>Click here to choose</option>
+<option value="Ink">Ink</option>
+<option value="Watercolour">Watercolour</option>
+<option value="Acrylic">Acrylic</option>
+<option value="Digital">Digital</option>
+</select>
+`
+
+const animationCathegories = `
+<select id="select">
+<option value="" selected disabled>Click here to choose</option>
+<option value="2D">2D</option>
+<option value="3D">3D</option>
+</select>
+`
+
+const showMessage = (message, sender) => {
+  if (sender === "user") {
+    chat.innerHTML += `
+      <section class="user-msg fade-in">
+        <div class="bubble user-bubble">
+          <p>${message}</p>
+        </div>
+        <img src="assets/user.png" alt="User" />  
+      </section>
+    `
+  } else if (sender === "bot") {
+    chat.innerHTML += `
+      <section class="bot-msg fade-in">
+        <img src="assets/bot.png" alt="Bot" />
+        <div class="bubble bot-bubble">
+          <p>${message}</p>
+        </div>
+      </section>
+    `
+  }
+  
+  setTimeout(() => {
+    document
+      .querySelectorAll(".fade-in")
+      .forEach((chatBubble) => {
+        chatBubble.classList.remove('fade-in')
```

Comment:
I realize that I mixed single and double quotes as well - my bad ;) Just a reminder to both of us then to be consistent:
`chatBubble.classList.remove("fade-in")`